Sources/Collection.swift

```diff
@@ -27,7 +27,7 @@ public final class Collection {
         mongoc_collection_destroy(self.pointer)
     }
 
-    public func insert(document: BSON.Document, flags: InsertFlags = .None) throws {
+    public func insert(document document: BSON.Document, flags: InsertFlags = .None) throws {
```

Comment:
Ok, removed—so weird, I don't know where it came from. Let me know what minor version to use for BinaryJSON, and I'll push an update.